src/Traits/CleansEnvFile.php

```diff
@@ -7,22 +7,35 @@
 
 trait CleansEnvFile
```

Comment:
NOTE: Added the overriding of the LOG env variables here. Committed this with wrong message